### DISCUSSION TITLE
quiero hacer una búsqueda por solo unos campos de una tabla pero no se en que estoy mal alguien me podría ayudar

### DISCUSSION THREAD

**Discussion by NimsiSanchez** [sequelize/sequelize#13822] [Help & Questions] [answered]
### **_-------------este es mi repositorio----------------------_**
async buscarEmpresa(params) {
        return this.db.tblEmpresa.findAll({
            //attributes: ['intClave', 'chrNombreETV', 'intFKIDEstatus'],
            where: { 
                intClave: params.intClave,
                chrNombreETV: params.chrNombreETV,
                intFKIDEstatus: params.intFKIDEstatus,
            },
        });
    }
**_### ----------------------este mi use case--------------------_**
async buscar(params){
        console.log(params);
        let response={}
        let paramsRes = {
            intClave: params.clave,
            chrNombreETV: params.nombre,
            intFKDIEstatus: params.estatus
        }
        const busqueda = await this.empresaRepository.buscarEmpresa(paramsRes);
        return busqueda
}

**Comment by fzn0x** [comment, marked as answer]
Kindly using english to get help by the international community :)
